### PR TITLE
First cleanup for info.xml docs

### PR DESCRIPTION
**Pull request by BernhardPosselt**
Ready to review

### CODE REVIEW

**Comment by BernhardPosselt** [comment]
@adsworth @LukasReschke @janibonnevier

File: requirements/base.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Django==1.9.7
-djangorestframework==3.3.3
+djangorestframework==3.4.0
 Markdown==2.6.6
```
Comment (janibonnevier) [inline requirements/base.txt]: Isn't markdown rendered with JS now?

Comment (BernhardPosselt) [inline requirements/base.txt]: removed markdown

**Comment by janibonnevier** [comment]
:+1:

**Pull request merged**